modules/distr-tracing-tempo-install-tempomonolithic-cli.adoc

```diff
@@ -6,10 +6,10 @@
 [id="distr-tracing-tempo-install-tempomonolithic-cli_{context}"]
```

Comment:
Callout 13 in the `Installing a TempoStack instance by using the CLI` and `Installing a TempoStack instance by using the web console` section needs a `TempoStack` reference in backticks.

Reply:
@kcarmichael08, thank you, I probably only saw the updated label and then didn't refresh the page.

Reply:
Thank you for spotting this, Darragh, I fixed this.